getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -161,6 +161,10 @@ located in the project root:
     </Project>
 
 
-Then using the dotnet command line to restore package, in the project root::
+Debugging Failing Godot Builds
+------------------------------
+
+If you find that your mono project is failing to build, you can debug more closely
+by running the following on the command line, in the root of your project::
 
-    $ dotnet restore
+    $ msbuild
```

Comment:
Why does this remove the last part of the nuget section?

Reply:
I don't understand what debugging it more closely means here, but the `dotnet restore` command was for restoring the dependencies. Although it may work, the correct command for us is `msbuild /t:restore`.

Reply:
Yeah, let me explain:

My intention for that last section was to not debug nuget restoring of packages, specifically. It was to provide the command to debug any failing mono build.

Currently in godot, a failed build results in a "mono failed to build" error, with no extra output in the logs or error (as far as I can tell). I was stuck for a while, and figured out that msbuild gives the debug output I was looking for. 

Previously I erroneously added dotnet/nuget restore because I thought it was necessary. In reality, as long as nuget has installed the packages somewhere for the project, msbuild does the right thing and references those packages. So I removed the "restore" command entirely. 

But you're right I accidentally removed the restore command that is required in the process. I'll push a fix.